xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodec.cpp

```diff
@@ -55,6 +55,8 @@ void VideoPicture::Reset()
 
   iWidth = 0;
   iHeight = 0;
+  m_xOffset = 0;
+  m_yOffset = 0;
```

Comment:
```suggestion
  iXOffset = 0;
  iYOffset = 0;
```

`VideoPicture` does not seem to use `m_` prefix for any other member, suggest you keep using the type prefix `i` for these new members.